Packs/Core/Integrations/CortexCoreIR/CortexCoreIR_test.py

```diff
@@ -366,3 +367,72 @@ def test_get_distribution_url_command_without_download_not_supported_type():
         get_distribution_url_command(client, args)
     client.get_distribution_url.assert_called_once_with("12345", "sh")
     assert e.value.message == "`download_package` argument can be used only for package_type 'x64' or 'x86'."
+
+# tests for core_execute_command_command
+def test_missing_command_raises():
```

Comment:
I would have called the function - 'test_core_execute_command_missing_command_raises'.
(relevant for all other tests as well).